doc/source/getting_started/comparison/comparison_with_stata.rst

```diff
@@ -2,7 +2,7 @@
 
 {{ header }}
 
-Comparison with Stata
+Comparison with STATA
```

Comment:
Can you revert this one please? Stata is capitalized `Stata` in their website, so let's use that.

Reply:
Sure! Thanks for pointing that out.